server/datastore/mysql/labels_test.go

```diff
@@ -323,6 +323,8 @@ func testLabelsListHostsInLabel(t *testing.T, db *Datastore) {
 
 	listHostsInLabelCheckCount(t, db, filter, l1.ID, fleet.HostListOptions{}, 3)
 
+	listHostsInLabelCheckCount(t, db, filter, l1.ID, fleet.HostListOptions{LowDiskSpaceFilter: ptr.Int(35), ListOptions: fleet.ListOptions{OrderKey: "id", After: "2"}}, 1)
```

Comment:
since we're testing OrderKey and After, we may want to assert that the correct rows are returned and that they're in the proper order

Reply:
Done!